microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,410 @@ ms.author: msedgedevrel
+The target app can't be set when using the environment variable approach.  Therefore, when set as a global environment, this setting will affect all apps that use WebView2 on the machine.
 
-   Replace `C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0` by the path to the desired Microsoft Edge preview channel.
 
-#### Resuming using the default, WebView2 Evergreen Runtime
+##### [Group policy](#tab/group-policy)
 
-To undo the above setting, run the following command:
+Set the `BrowserExecutableFolder` policy.
 
-`REG DELETE HKLM\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /f`
+Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`  <!-- 2 trailing spaces = linebreak -->
+Value: `<path to desired Microsoft Edge preview channel>`
 
-### Registry override: release channel preference
+Do either of the following:
 
-To make your application use a Microsoft Edge preview channel by using a registry override that changes the release channel preference by changing the order of searching for a channel:
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download).  See [Configure Microsoft Edge policy settings on Windows devices](/deployedge/configure-microsoft-edge).
+
+* Use the built-in policy on Intune.  See [Configure Microsoft Edge policy settings in Microsoft Intune](/mem/intune/configuration/administrative-templates-configure-edge).
+
+---
+
+
+<!-- ====================================================================== -->
+## FAQs
+
+
+<!-- ------------------------------ -->
+#### When does the new runtime get picked up?
+
+If you are using an environment variable, registry key, or group policy, they are only applied to any new WebView2 Runtime browser processes that are created; they do not apply to WebView2 Runtime browser processes that are already running.  For example, if you already have a WebView2 with a specific user data folder (and therefore a corresponding browser process) and change `ChannelSearchKind` before attempting to create a new WebView2 using that same user data folder, it will use the existing WebView2 Runtime browser process, and not search for a different runtime.
+
+To ensure that all processes use the environment variable, registry key, or group policy, you need to do any of the following:
+* Restart the app.
+* Re-create the WebView2 controls.
+* Create a new WebView2 that has a different user data folder than any other currently running WebView2 controls.
+
+
+<!-- ------------------------------ -->
+#### When using `ChannelSearchKind`, how do I make sure the right channel is picked up when I have multiple preview browser channels?
```

Comment:
This is a really big section that feels like it doesn't belong in the FAQ. Instead we should probably have a link above in the "Channel Search Kind" that's like "If you need a more specific order, checkout this section + <link>".

Reply:
trying latest heading:
```
## How to use `ChannelSearchKind` to ensure that the correct preview browser channel is used
```

Reply:
Link from which section to which section?  Outline of article:
```
# Test upcoming APIs and features
#### Use a preview channel of WebView2 to test your app against the most recent experimental WebView2 APIs
#### Interplay between the WebView2 Prerelease SDK and Microsoft Edge Preview Channels
#### Forward-compatibility testing by using the preview channel

## Downloading the prerelease SDK and a preview channel

## Approaches to making your app use a specific browser channel

## Switching the channel search order (recommended)
##### [API](#tab/api)
##### [Registry key](#tab/registry-key)
##### [Environment variable](#tab/environment-variable)
##### [Group policy](#tab/group-policy)

## Setting the browser executable folder (for local testing)
##### [API](#tab/api)
##### [Registry key](#tab/registry-key)
##### [Environment variable](#tab/environment-variable)
##### [Group policy](#tab/group-policy)

## When the new runtime gets picked up

## How to use `ChannelSearchKind` to ensure that a particular preview browser channel is used
##### [API](#tab/api)
##### [Registry key](#tab/registry-key)
##### [Environment variable](#tab/environment-variable)
##### [Group policy](#tab/group-policy)

## How to set an environment variable
#### Setting environmental variables by using APIs, for a specific app
#### Setting environmental variables globally, for all apps

## Deploy preview channels
#### Option 1: Manually install preview channels on machines
#### Option 2: Programmatically deploy preview channels via API endpoints

## See also
```
@champnic @victorhuangwq

Reply:
new outline after moved the section up:
```
# Test upcoming APIs and features
#### Use a preview channel of WebView2 to test your app against the most recent experimental WebView2 APIs
#### Interplay between the WebView2 Prerelease SDK and Microsoft Edge Preview Channels
#### Forward-compatibility testing by using the preview channel

## Downloading the prerelease SDK and a preview channel

## Approaches to making your app use a specific browser channel

## Switching the channel search order (recommended)
##### [API](#tab/api)
##### [Registry key](#tab/registry-key)
##### [Environment variable](#tab/environment-variable)
##### [Group policy](#tab/group-policy)

## How to use `ChannelSearchKind` to ensure that a particular preview browser channel is used
##### [API](#tab/api)
##### [Registry key](#tab/registry-key)
##### [Environment variable](#tab/environment-variable)
##### [Group policy](#tab/group-policy)

## Setting the browser executable folder (for local testing)
##### [API](#tab/api)
##### [Registry key](#tab/registry-key)
##### [Environment variable](#tab/environment-variable)
##### [Group policy](#tab/group-policy)

## When the new runtime gets picked up

## How to set an environment variable
#### Setting environmental variables by using APIs, for a specific app
#### Setting environmental variables globally, for all apps

## Deploy preview channels
#### Option 1: Manually install preview channels on machines
#### Option 2: Programmatically deploy preview channels via API endpoints

## See also
```